contrib/meson.build

```diff
@@ -5,13 +5,13 @@ glib_dep = dependency('glib-2.0')
 m_dep = cc.find_library('m', required: false)
 capng_dep = dependency('libcap-ng', required: get_option('capng'))
 ncurses_dep = dependency('curses', required: get_option('ui'))
-numa_dep = dependency('numa')
+numa_dep = dependency('numa', required: get_option('numa'))
```

Comment:
So, i've obviously tipped my hand previously.  I'm not near as familiar with meson as I am with autotools.  These look like a pretty straightforward mapping of autotools options to meson build options, but in autotools --with-numa is the default (ditto --with-irqbalance-ui).  Shouldn't these be defaulted to on, with an option to disable?  Or is the default value set somewhere that I'm not seeing?

Reply:
>  I'm not near as familiar with meson as I am with autotools.

Well, I'm not judging :)

>  Shouldn't these be defaulted to on, with an option to disable? Or is the default value set somewhere that I'm not seeing?

These are configured in meson_options.txt, representing the [build options](https://mesonbuild.com/Build-options.html#build-options) for the project. In our particular case here we have [features](https://mesonbuild.com/Build-options.html#features), which can be `auto`, `enabled` or `disabled` (when building you would e.g. pass `-Dthermal=enabled` to `meson` and downstream on Arch Linux we in fact make use of `--auto-features enabled`, which switches all features to `enabled`).

Currently, the `meson_options.txt` does not specify any default and I think this means to default to `auto` which is apparently the same as saying: `required: false`, when the `required: get_option('numa')` is resolved.

Reply:
Can it be changed so that numa is defaulted to enabled and we can disable it on the command line?

Reply:
Yes. This can be added in the meson_options.txt.
I'll adapt it tomorrow! :)

Reply:
Sorry, took me much longer than anticipated, as I fell ill last week.

I have now adapted the `meson_options.txt` (should be renamed to `meson.options` in the future) to enable the NUMA feature by default.